src/NServiceBus.AcceptanceTests/Sagas/When_finder_exists_and_found_saga.cs

```diff
@@ -11,14 +11,18 @@
     using Persistence;
 
     [TestFixture]
-    public class When_a_finder_exists_and_found_saga : NServiceBusAcceptanceTest
+    public class When_finder_exists_and_found_saga : NServiceBusAcceptanceTest
```

Comment:
Test name and method doesn't line up, how about:

class When_using_finder_for_existing_instance.Should_find_instance() ?